SwiftUIHomeworks/SwiftUIHomeworks/ViewModels/PurchaseViewModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  PurchaseViewModel.swift
+//  SwiftUIHomeworks
+//
+//  Created by Aleksandr Dorofeev on 22.01.2023.
+//
+
+import Foundation
```

Comment:
```suggestion
import Foundation
```
разве работает без import SwiftUI?)))

Reply:
да) А без Foundation не работает)